packages/core/analytics/core-analytics-browser-internal/src/analytics_service.ts

```diff
@@ -34,6 +35,8 @@ export class AnalyticsService {
     });
 
     this.registerBuildInfoAnalyticsContext(core);
+    // Register special `metrics` type
+    registerMetricEventType(this.analyticsClient);
```

Comment:
The analytics service in core registers the metric type 😇 

We can work on a follow up PR to move the `reportMetricEvent` API to an actual API exposed by the AnalyticServices in core. What do you think?

Reply:
Don't think it's critical, we already have a package.
As you fish 🐟